Gems/ROS2/Code/Source/Manipulation/JointsTrajectoryComponent.h

```diff
@@ -61,7 +61,6 @@ namespace ROS2
         TrajectoryGoal m_trajectoryGoal;
         rclcpp::Time m_trajectoryExecutionStartTime;
 
-        JointsManipulationRequests::ManipulationJoints m_manipulationJoints; //!< Joints: assumes they don't change dynamically.
```

Comment:
Technically we can dynamically change non-articulated robots, so I would not assume. With implementation from #366 we initialized this field to empty in JointsManipulationRequests::Activation.